packages/js/assets/user-feedback-form.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * todo: this js file (bundled with the html file) should either:
```

Comment:
Yup, I'm fine with creating a server-side endpoint for serving the JS

Reply:
Is there a reason why this JS would need to be served from API instances rather than from our JS CDN?

Reply:
Good idea! I can't think of a reason at the moment.
We could use the JS CDN. How does it work though? How can I add this as a resource/route?

Reply:
Thanks for the suggestion! I am uploading the script to our CDN! I think it's better because the js client has control over implementation + distribution.